estuary/api/v1.py

```diff
@@ -240,14 +242,14 @@ def get_siblings(resource, uid):
     :raises NotFound: if the item is not found
     :raises ValidationError: if an invalid resource was requested
     """
+    story_manager_arg = request.args.get('story', 'ContainerStoryManager')
     # This is the node that is part of a story which has the relationship to the desired
     # sibling nodes
     story_node = get_neo4j_node(resource, uid)
     if not story_node:
         raise NotFound('This item does not exist')
-
     # Defaulting to Container Story until we modify this endpoint to suit the module changes
-    story_manager = ContainerStoryManager()
+    story_manager = globals().get(story_manager_arg)()
```

Comment:
This is quite dangerous because this allows the user to pass in a string of any function or class name and cause the back-end to execute it. My suggestion is that the argument should be something like `story_type` and then the back-end tries to find the matching story manager class. An example of input would be `container` or `module`.

Reply:
yeah, that makes sense. I will change it.